modules/ROOT/pages/to-download-and-install-studio-lx.adoc

```diff
@@ -3,27 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+== Prerequisites
 
-* Before downloading and launching Anypoint Studio, make sure you have the Java SE JDK 8 installed.
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
```

Comment:
To support some Anypoint Studio Themes in Linux, have GTX v2 installed.

(Avoid "it is recommended," which is passive but also I think that we want to sound as if everything we document is our recommended best practice.)

Reply:
Question: 
I'm not sure that I understand "some of Studio Themes in Linux." Does this mean that you can use only some of the themes in a Linux environment, or that some of the themes require GTK v2 if you are working in a Linux environment, but some do not (all are supported in Linux, but some need GTK)?

Reply:
Also, is the audience already familiar with GTK? If not, suggest spelling GIMP Toolkit or General Image Manipulation Program (GIMP) Toolkit.

Reply:
GTK is a widely known term in Linux. 
We had users using Ubuntu with other versions of GTK that had problems visualizing some behaviors such as pop-up windows.
I think I'll change that to 
```
To support the Studio Theme in Linux, have GTK version 2 installed.
```